pkgs/by-name/me/mealie/package.nix

```diff
@@ -0,0 +1,160 @@
+{ lib
+, callPackage
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, python3
```

Comment:
Use `python3Packages` instead. Apparently it works better for cross compilation. It maps to `python3.pkgs`.

Reply:
I do not only need `python3.pkgs`, I also require `python3.sitePackages`, to do an override of `packageOverrides` for pydantic, and call `python3.interpreter`

Reply:
Then use `python3Packages.python.sitePackages`.

Reply:
Thanks, fixed it